src/fastaudio/augment/signal.py

```diff
@@ -120,14 +120,15 @@ def encodes(self, sg: AudioSpectrogram):
         return shift_signal(sg, int(s), self.roll)
 
 
-class AddNoise(Transform):
+class AddNoise(RandTransform):
     "Adds noise of specified color and level to the audio signal"
 
-    def __init__(self, noise_level=0.05, color=NoiseColor.White):
+    def __init__(self, p=0.5, noise_level=0.05, color=NoiseColor.White):
```

Comment:
It will be better if this `p` parameter is moved to the end of the init signature. Otherwise, if someone is creating a transform like `AddNoise(0.1)`, having `p` as the first argument will cause a silent breaking change